docs/install.rst

```diff
@@ -10,7 +10,7 @@ Installing Kanister
 Prerequisites
 =============
 
-* Kubernetes 1.8 or higher with Beta APIs enabled
+* Kubernetes 1.16 or higher
```

Comment:
Should we add a note saying they can install 0.60.0 in case they want < 1.15 ?

Reply:
right, we should do that. I will make the changes.